test/test_nn.py

```diff
@@ -10794,6 +10795,7 @@ def test_ctc_loss_cudnn(self, device):
         self.assertEqual(grad_cudnn, grad_native, atol=1e-4, rtol=0)
 
     @dtypesIfCUDA(torch.half, torch.float, torch.double)
+    @skipIfMps
```

Comment:
Can you please add a comment with the reason it's crashing? 
I think if you could do something like `skipMPSIf("Reason")` that would be good (see skipCudaIf). Also I think there are bunch of skipIfMPS in this file that are not needed
By default the floating point dtype in this file is set to double. I think this change got overridden by the recent rebase, but we'd need it to be float for mps (see https://github.com/kulinseth/pytorch/pull/294/files#diff-7927db349f568afca2de9b94d74ea5c3b8cb468cb6a433d0cc1e61e65c515a36R21)

Reply:
I think we also need to get rid of all the skipIfMPS that are currently in and check how it goes with `float` dtype